src/installation/install-linux-opensuse.md

```diff
@@ -0,0 +1,83 @@
+# Linux (OpenSUSE)
+
+This was verified working on Leap 15 and should be fine with Tumbleweed as well but has not been independatly verified.
+
+## 1 Update your repositories and packages fully:
+
+```bash
+sudo zypper dup
+```
+
+## 2 Install necessary dependencies:
+
+```bash    
+sudo zypper install nodejs gcc make git`
+```
+
+## 3 Verify correct Nodejs version (needs to be 14+)
+
+```bash
+node --version`
+```
+
+## 4 Import the necessary pgp key and add the MongoDB community repository
+
+```bash
+sudo rpm --import https://www.mongodb.org/static/pgp/server-5.0.asc
+sudo zypper addrepo --gpgcheck "https://repo.mongodb.org/zypper/suse/15/mongodb-org/5.0/x86_64/" mongodb
+```
+ 
+## 5 Install, start and make MongoDB persist reboots:
+
+```bash
+sudo zypper install mongodb-org
+sudo systemctl enable mongod --now
+sudo systemctl status mongod
+```
+    
+## 6 Install pm2:
+
+```bash
+sudo npm install pm2 -g
+```
+
+## 7 Obtain and start OctoFarm
+
+```bash
+git clone --depth 1 https://github.com/OctoFarm/OctoFarm.git
+cd OctoFarm/
+npm start
+pm2 list
+```
+
+- You should see something similar to:
+
+```mnavarro@PaintedGreen:~> pm2 list
+┌────┬────────────────────┬──────────┬──────┬───────────┬──────────┬──────────┐
+│ id │ name               │ mode     │ ↺    │ status    │ cpu      │ memory   │
+├────┼────────────────────┼──────────┼──────┼───────────┼──────────┼──────────┤
+│ 0  │ OctoFarm           │ fork     │ 0    │ online    │ 0%       │ 239.6mb  │
+└────┴────────────────────┴──────────┴──────┴───────────┴──────────┴──────────┘
+```
+
+## 8 Allow traffic on port 4000 during the current boot:
+
+- By default OctoFarm runs on port 4000, adjust as required:
+
+```bash
+sudo firewall-cmd --add-port=4000/tcp
+sudo firewall-cmd --add-port=4000/udp
+```
+
+## 9 And finally with one additional flag to make it persist reboots
+
+```bash
+    sudo firewall-cmd --add-port=4000/udp --permanent
+    sudo firewall-cmd --add-port=4000/tcp --permanent
+```
+
+That's it. Navigate to <printer_ip>:4000 (example: "192.168.100.10:4000") in your browser and have a day!
```

Comment:
Should this be "have a nice day!" ?

Reply:
Private joke of mine I suppose. I find it puts a lot of pressure on people to put a qualifier on their day; it is enough for me that they are having a day. I am happy to change it if it is confusing or out of place, just let me know!

Michael

Reply:
No I'm laughing about it now ahaha I'll keep it :) consider it accepted so I'll merge and start build :) big thanks again.